### PR TITLE
Update view to reflect the new detection table structure

### PR DESCRIPTION
**Pull request by jkburges**

### CODE REVIEW

File: grails-app/migrations/join_dets_and_rxrs.groovy [hunk 1]
```diff
@@ -272,4 +272,57 @@ databaseChangeLog = {
...
+    changeSet(author: "jburgess", id: "1430268900000-09") {
+        createView("""
+            SELECT
+              station_station_name AS station,
+              installation_name AS installation,
+              rxr_project_name AS project,
+              aatams.scramblepoint(st_makepoint(station_longitude, station_latitude)) AS public_location,
+              st_x(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lon,
+              st_y(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lat,
+              ('http://localhost:8080/aatams/installationStation/show/'::text || station_id) AS installation_station_url,
```
Comment (dnahodil) [inline grails-app/migrations/join_dets_and_rxrs.groovy]: Is this meant to have URLs baked-in?

If so, why are they local URLs?

File: grails-app/migrations/join_dets_and_rxrs.groovy [hunk 2]
```diff
@@ -272,4 +272,57 @@ databaseChangeLog = {
...
+              ('http://localhost:8080/aatams/detection/lISt?filter.receiverDeployment.station.in=name&filter.receiverDeployment.station.in='::text || (station_station_name)::text) AS detection_download_url,
+              count(*) AS detection_count,
+              ((log((GREATEST(count(*), (1)::bigint))::double precISion) / log(((SELECT max(t.detection_count) AS max FROM (
+                SELECT
+                  station_station_name,
+                  count(station_station_name) AS detection_count
+                FROM aatams.detection_view GROUP BY station_station_name
+                ) t))::double precISion)) * (10)::double precISion) AS relative_detection_count,
```
Comment (dnahodil) [inline grails-app/migrations/join_dets_and_rxrs.groovy]: The capitalisation in 'precision' is cray-cray. Is that a real thing, or a copy-and-paste error?

File: grails-app/migrations/join_dets_and_rxrs.groovy (hunk 1 above)
Comment (jkburges) [inline grails-app/migrations/join_dets_and_rxrs.groovy]: TBH, I wondered the same thing.  Take a look at https://github.com/aodn/aatams/blob/38e5f8cec160d2790673796cd33fe65a3b5f3d3b/grails-app/migrations/rebased_db_initialisation.groovy#L2134 though, and you will see that this is (hopefully) equivalent to what was there before.  I didn't want to change it without investigating...  Might be worth raising an issue on this one though, seems wrong to me.

File: grails-app/migrations/join_dets_and_rxrs.groovy (hunk 2 above)
Comment (jkburges) [inline grails-app/migrations/join_dets_and_rxrs.groovy]: Fixed in b9f7d5b46e64

File: grails-app/migrations/join_dets_and_rxrs.groovy (hunk 1 above)
Comment (dnahodil) [inline grails-app/migrations/join_dets_and_rxrs.groovy]: Ah, I see that it's maintained the existing behaviour. I've added a ticket on the board for us to check it. If it is broken we can look into it.

**Comment by dnahodil** [comment]
Lol @ commit message on b9f7d5b

**Comment by jkburges** [comment]
Is there something left for me to do here?

**Comment by dnahodil** [comment]
Nah, I'm just trying to get my AATAMS environment back up to see this new view in the db.

**Comment by dnahodil** [comment]
Just looking at a failing test

**Pull request merged**